scripts/vertex_mock_responses.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright 2024 Google LLC
```

Comment:
Is this script forked from google repo or written from scratch? If this one is not forked then probably should apply invertase's license header?